tests/scripts/t3000/run_t3000_perplexity_tests.sh

```diff
@@ -148,12 +148,12 @@ run_t3000_llama3_perplexity_tests_t3000() {
 
   for MESH_DEVICE in T3K; do
     for LLAMA_DIR in "$llama1b" "$llama3b" "$llama8b" "$llama11b"; do
-      MESH_DEVICE=$MESH_DEVICE LLAMA_DIR=$LLAMA_DIR pytest -n auto models/tt_transformers/tests/test_accuracy.py --timeout=3600 ; fail+=$?
+      MESH_DEVICE=$MESH_DEVICE LLAMA_DIR=$LLAMA_DIR pytest -n auto models/tt_transformers/demo/simple_text_demo.py -k ci-token-matching --timeout=3600 ; fail+=$?
     done
 
     # 70B and 90B tests has the same configuration between `-k "attention-accuracy"` and `-k "attention-performance"` so we only run one of them
     for LLAMA_DIR in "$llama70b" "$llama90b"; do
-      MESH_DEVICE=$MESH_DEVICE LLAMA_DIR=$LLAMA_DIR pytest -n auto models/tt_transformers/tests/test_accuracy.py -k "attention-accuracy" --timeout=3600 ; fail+=$?
+      MESH_DEVICE=$MESH_DEVICE LLAMA_DIR=$LLAMA_DIR pytest -n auto models/tt_transformers/demo/simple_text_demo.py -k "performance and ci-token-matching" --timeout=3600 ; fail+=$?
```

Comment:
I would use one of the suggested keyword in the comment above. I remember seeing a weird issue with id resolution and longer id pattern matching solved it.
```suggestion
      MESH_DEVICE=$MESH_DEVICE LLAMA_DIR=$LLAMA_DIR pytest -n auto models/tt_transformers/demo/simple_text_demo.py -k "attention-accuracy and ci-token-matching" --timeout=3600 ; fail+=$?
```